components/Btn/Btn.css

```diff
@@ -3,7 +3,7 @@
   font-family: var(--font-brandon);
   font-size: 1em;
   font-weight: 400;
-  padding: 0.5em 1em;
+  padding: 0.5em 1.25em;
```

Comment:
Well it means that _all_ buttons will be slightly bigger. Mostly curious to see how it looks on the review app